run/idp-sql/main.py

```diff
@@ -97,11 +97,17 @@ def shutdown_handler(signal: int, frame: FrameType) -> None:
     sys.exit(0)
 
 
+# [END cloudrun_sigterm_handler]
+
 if __name__ == "__main__":
     # handles Ctrl-C locally
     signal.signal(signal.SIGINT, shutdown_handler)
 
     app.run(host="127.0.0.1", port=8080, debug=True)
 else:
     # handles Cloud Run container termination
+    # [START cloudrun_sigterm_handler]
```

Comment:
I'd expect the code comment for a statement be included with the statement.

Do we want to include the comments for the statements here and above to include the comment?

Reply:
I specifically excluded them since they aren't too helpful out of context.